vars/perfRegressionParallelPipeline.groovy

```diff
@@ -37,6 +38,14 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('post_behavior_monitor_nodes', 'keep-on-failure')}",
                    description: 'keep|keep-on-failure|destroy',
                    name: 'post_behavior_monitor_nodes')
+
+            string(defaultValue: "${groovy.json.JsonOutput.toJson(pipelineParams.get('sub_tests', ["test_write"]))}",
```

Comment:
why only "test_write"?

Reply:
Which performance subtest to use as default? or nothing to use and this have to be required field?

Reply:
Maybe empty, and add check that would fail with clear message

Reply:
why not list of all of them? doesn't it run all sub_tests in parallel?

Reply:
yes, all subtests run in parallel.

Reply:
@fruch @roydahan 
Added next logic. default value for field is what defined in *.jenkinsfile. if subtests are not defined in *.jenkinsfile, they could be defined via job param. 
if sub_tests is not defined, then executed  test defined in *.jenkins test_name - all test method will execute consistently.

Reply:
to run in parallel, sub_tests should be defined